clients/common/rocblas_gentest.py

```diff
@@ -211,12 +211,16 @@ def setdefaults(test):
                              test['K']))
             test.setdefault('stride_c', test['ldc'] * test['N'])
             test.setdefault('stride_d', test['ldd'] * test['N'])
+            test.setdefault('stride_x', 0)
+            test.setdefault('stride_y', 0)
             return
 
     test.setdefault('stride_a', 0)
     test.setdefault('stride_b', 0)
     test.setdefault('stride_c', 0)
     test.setdefault('stride_d', 0)
+    test.setdefault('stride_x', 0)
+    test.setdefault('stride_y', 0)
```

Comment:
The defaults should be set in `rocblas_common.yaml` instead of `rocblas_gentest.py` if they are constant.

Please move them to `rocblas_common.yaml`.

Reply:
Ok. Moving them...